README.md

```diff
@@ -145,14 +146,14 @@ module "collector_event_hub" {
 | Name | Description |
 |------|-------------|
 | <a name="output_nsg_id"></a> [nsg\_id](#output\_nsg\_id) | ID of the network security group attached to the Collector Server nodes |
+| <a name="output_user_data"></a> [user\_data](#output\_user\_data) | n/a |
```

Comment:
I don't see any change to `outputs.tf` so wondering if this is an element of debug testing?

Reply:
Yup, an artefact of my debugging. Thanks for catching, fixed!